### PR TITLE
Add popup on first launch

### PR DESCRIPTION
**Pull request by mattie078**

### CODE REVIEW

File: app/assets/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    background: rgba(0, 0, 0, 0.5);
+  }
+  
+  /* Main Popup content. */
+  #popupContent {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /*justify-content: space-between;*/
```
Comment (Brend-Smits) [inline app/assets/css/custom.css]: Remove commented out code

File: app/assets/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    width: 400px;
+    /*height: 35%;*/
```
Comment (Brend-Smits) [inline app/assets/css/custom.css]: Remove commented out code

File: app/assets/css/custom.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    box-sizing: border-box;
+    padding: 15px 0px;
+    /* background-color: #424242; */
```
Comment (Brend-Smits) [inline app/assets/css/custom.css]: Remove commented out code

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
Looks great, thanks a lot :) Are you interested in working on more features for the launcher?

**Comment by mattie078** [comment]
> Looks great, thanks a lot :) Are you interested in working on more features for the launcher?

Thanks!
Yeah sounds good, though this is also my first time working Electron.

**Pull request merged**